doc/source/rllib/rllib-examples.rst

```diff
@@ -140,7 +140,7 @@ Serving and Offline
 - `Saving experiences <https://github.com/ray-project/ray/blob/master/rllib/examples/saving_experiences.py>`__:
    Example of how to externally generate experience batches in RLlib-compatible format.
 - `Finding a checkpoint using custom criteria <https://github.com/ray-project/ray/blob/master/rllib/examples/checkpoint_by_custom_criteria.py>`__:
-   Example of how to find a checkpoint after a `Tuner.fit()` via some custom defined criteria.
+   Example of how to find a `checkpoint <rllib-saving-and-loading-algos-and-policies.html>`__ after a `Tuner.fit()` via some custom defined criteria.
```

Comment:
not a big deal, but using a `:ref:` here is more stable and prevents outdated links when moving files etc.

Reply:
Done, but can you check, whether I did this correctly. Not sure how sphinx infers the actual html file. E.g. I don't find any `serve-rllib-tutorial` either (referenced further above this one here).

Reply:
@sven1977 yeah, I don't like the way Sphinx solves this. essentially, if there's a file called `serve-rllib-tutorial.md` in the source, relative to the same file, you can use that as doc ref, otherwise (and this is usually much better), you'd just add a `_ serve-rllib-tutorial ::` tag in rst or `(serve-rllib-tutorial)=` in markdown in the respective doc and reference that, see here:

https://raw.githubusercontent.com/ray-project/ray/23b3a599b9df8100558c477e94b0b19b1a38ac27/doc/source/serve/tutorials/rllib.md

Reply:
so in this case, please put `.. _rllib-saving-and-loading-algos:` at the top of your  new `doc/source/rllib/rllib-saving-and-loading-algos-and-policies.rst` file. then you can reference it as:

```
:ref:`my reference <rllib-saving-and-loading-algos>`
```

this should also fix the build